operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -28,27 +28,31 @@ function check_for_shutdown() {
   state=`${SCRIPTPATH}/readState.sh`
   exit_status=$?
   if [ $exit_status -ne 0 ]; then
-    trace "Node manager not running or server instance not found; assuming shutdown" &>> /weblogic-operator/stopserver.out
+    trace "Node manager not running or server instance not found; assuming shutdown" &>> ${SERVER_OUT_FILE}
     return 0
   fi
 
   if [ "$state" = "SHUTDOWN" ]; then
-    trace "Server is shutdown" &>> /weblogic-operator/stopserver.out
+    trace "Server is shutdown" &>> ${SERVER_OUT_FILE}
     return 0
   fi
 
   if [[ "$state" =~ ^FAILED ]]; then
-    trace "Server in failed state" &>> /weblogic-operator/stopserver.out
+    trace "Server in failed state" &>> ${SERVER_OUT_FILE}
     return 0
   fi
 
-  trace "Server is currently in state $state" &>> /weblogic-operator/stopserver.out
+  trace "Server is currently in state $state" &>> ${SERVER_OUT_FILE}
   return 1
 }
 
+# setup ".out" location for a WL server
+serverLogHome="${LOG_HOME:-${DOMAIN_HOME}/servers/${SERVER_NAME}/logs}"
+SERVER_OUT_FILE="${serverLogHome}/stop-${SERVER_NAME}.out"
```

Comment:
Two comments:
- Can you use a different name for the env var?  Maybe 'STOP_OUT_FILE'?  The name SERVER_OUT_FILE is already used for other purposes in other scripts.
- Consider starting the out file name with ${SERVER_NAME} so that it sorts next to other .log and .out files that are generated for the particular server...

Reply:
I made the changes

Reply:
Ok. Resolving.